### PR TITLE
Automate setting milestone for servicing PRs when merged

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
The accompanying changer for the process is also being updated: https://github.com/dotnet/aspnetcore-internal/pull/4187

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
@dotnet/aspnet-build folks, do we have an issue tracking work to not run CI checks when non-code file is being touched (at least not related to functionality). This one is a great example.

**Comment by wtgodbe** [comment]
> do we have an issue tracking work to not run CI checks when non-code file is being touched (at least not related to functionality). This one is a great example.

We did that for a but, but aspnetcore-ci is a required check - if we skip it, github sees it didn't succeed, and therefore won't allow the PR to be merged except by someone with admin rights. We decided that was more of a headache than running CI on these PRs, so we reverted the behavior.

**Review by wtgodbe — APPROVED** [review]
LGTM, is there a corresponding PR in aspnetcore-internal to update our checklist for branding activities?

File: .github/fabricbot.json [hunk 1]
```diff
@@ -3361,6 +3361,126 @@
...
+          "pull_request",
+          "issues",
+          "project_card"
+        ],
+        "taskName": "[Milestone Assignments] Assign Milestone to PRs merged to release/7.0 branch",
+        "actions": [
+          {
+            "name": "addMilestone",
+            "parameters": {
+              "milestoneName": "7.0.3"
```
Comment (dougbu) [inline .github/fabricbot.json:3479]: For a few days (until next branding event)&hellip;
```suggestion
              "milestoneName": "7.0.2"
```

**Review by dougbu — APPROVED** [review]
Needs a 1-line update but fine to merge w/o further review

**Comment by mkArtakMSFT** [comment]
> LGTM, is there a corresponding PR in aspnetcore-internal to update our checklist for branding activities?

See description.

**Review by dougbu — APPROVED** [review]

**Comment by mkArtakMSFT** [comment]
Thanks @dougbu and @wtgodbe!
Force-merging, as there are no code changes involved.

**Pull request merged**